django/contrib/gis/db/backends/oracle/features.py

```diff
@@ -12,3 +13,13 @@ class DatabaseFeatures(BaseSpatialFeatures, OracleDatabaseFeatures):
     supports_dwithin_distance_expr = False
     supports_tolerance_parameter = True
     unsupported_geojson_options = {'bbox', 'crs', 'precision'}
+
+    @cached_property
+    def django_test_skips(self):
+        skips = super().django_test_skips
+        skips.update({
+            "Oracle doesn't support spatial operators in constraints.": {
+                'gis_tests.gis_migrations.test_operations.OperationTests.test_add_check_constraint',
+            },
```

Comment:
It seems that spatial operators cannot be used in constraints, see [_"Usage notes"_](https://docs.oracle.com/en/database/oracle/oracle-database/19/spatl/spatial-operators-reference.html#GUID-97C17C18-F05E-49B4-BE11-E89B972E2A02):

> _"The operator must always be used in a **WHERE** clause, ..."_

@jtiai Jani, Can you take a look?

Reply:
I think that only applies to `SDO_FILTER` but I never recall seeing operators being used in constraints.

Reply:
I tried to use a generated condition the `WHERE` clause and it works fine, so IMO we can safely assume that it's an Oracle caveat :smile:

Reply:
Thanks!